eth/ethtest/utils_test.go

```diff
@@ -0,0 +1,300 @@
+package ethtest
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"github.com/bloxapp/ssv/ekm"
+	"github.com/bloxapp/ssv/eth/contract"
+	"github.com/bloxapp/ssv/eth/eventhandler"
+	"github.com/bloxapp/ssv/eth/eventparser"
+	"github.com/bloxapp/ssv/eth/simulator"
+	ibftstorage "github.com/bloxapp/ssv/ibft/storage"
+	"github.com/bloxapp/ssv/networkconfig"
+	operatorstorage "github.com/bloxapp/ssv/operator/storage"
+	"github.com/bloxapp/ssv/operator/validator"
+	"github.com/bloxapp/ssv/operator/validator/mocks"
+	"github.com/bloxapp/ssv/protocol/v2/blockchain/beacon"
+	registrystorage "github.com/bloxapp/ssv/registry/storage"
+	"github.com/bloxapp/ssv/storage/basedb"
+	"github.com/bloxapp/ssv/storage/kv"
+	"github.com/bloxapp/ssv/utils/blskeygen"
+	"github.com/bloxapp/ssv/utils/rsaencryption"
+	"github.com/bloxapp/ssv/utils/threshold"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/golang/mock/gomock"
+	"github.com/herumi/bls-eth-go-binary/bls"
+	"go.uber.org/zap"
+	"math/big"
+
+	"testing"
+)
+
+type testValidatorData struct {
+	masterKey        *bls.SecretKey
+	masterPubKey     *bls.PublicKey
+	masterPublicKeys bls.PublicKeys
+	operatorsShares  []*testShare
+}
+
+type testOperator struct {
+	id   uint64
+	pub  []byte // rsa pub
```

Comment:
If this needs to be commented, I'd consider renaming the struct fields, rsaPub/rsaPriv or rsaPK/rsaSK. But I'd rather remove comments

Reply:
Agree. Fixed